README.md

```diff
@@ -7,916 +7,62 @@
-    #     http://<ip-address>:3100/loki/api/v1/push
-    location: <url>
-
-    # (Optional) A list of services whose logs will be sent to this target.
-    # Use the special keyword 'all' to match all services in the plan.
-    # When merging log targets, the 'services' lists are appended. Prefix a
-    # service name with a minus (e.g. '-svc1') to remove a previously added
-    # service. '-all' will remove all services.
-    services: [<service names>]
-
-    # (Optional) A list of key/value pairs defining labels which should be set
-    # on the outgoing logs. The label values may contain $ENV_VARS, which will
-    # be substituted using the environment for the corresponding service.
-    labels:
-      <label name>: <label value>
-```
-
-## API and clients
-
-The Pebble daemon exposes an API (HTTP over a unix socket) to allow remote clients to interact with the daemon. It can start and stop services, add configuration layers the plan, and so on.
+You can also follow the [Getting started with Pebble tutorial](https://canonical-pebble.readthedocs-hosted.com/en/latest/tutorial/getting-started/).
 
-There is currently no official documentation for the API at the HTTP level (apart from the [code itself](https://github.com/canonical/pebble/blob/master/internals/daemon/api.go)!); most users will interact with it via the Pebble command line interface or by using the Go or Python clients.
+## Getting help
 
-The Go client is used primarily by the CLI, but is importable and can be used by other tools too. See the [reference documentation and examples](https://pkg.go.dev/github.com/canonical/pebble/client) at pkg.go.dev.
+If you need support, start with the [documentation](https://canonical-pebble.readthedocs-hosted.com/en/latest/).
 
-We try to never change the underlying HTTP API in a backwards-incompatible way, however, in rare cases we may change the Go client in a backwards-incompatible way.
+To learn more about Pebble, read the following sections:
 
-In addition to the Go client, there's also a [Python client](https://github.com/canonical/operator/blob/master/ops/pebble.py) for the Pebble API that's part of the [`ops` library](https://github.com/canonical/operator) used by Juju charms ([documentation here](https://juju.is/docs/sdk/interact-with-pebble)).
+- [General model](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/general-model/)
+- [Layer configuration examples](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
+- [Container usage](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
+- [Layer specification](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layer-specification/)
+- [API and clients](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/)
+- [Hacking / Development](#hacking--development)
+- [Contributing](#contributing)
 
-## Hacking / Development
+## Hacking / development
```

Comment:
Maybe `Hacking and development`

Reply:
I also reckon we should remove the "Have fun! and enjoy the rest of the year!" section ... just seems a bit weird to me.